towers/clio/level_004.rb

```diff
@@ -1,15 +1,24 @@
-#  ---
-# |@ >|
-#  ---
+#  -------
+# |    .  |
+# |@   G >|
```

Comment:
Oops looks like it's merged already, but this should be a "g"

Reply:
You're right. That said, this commented out map is only useful to the implementer. When you run the levels, the README map is generated from the code, it's not taken from these comments.
(which is nice as it allows you to see where you're placing invisible units like the traps, but they are not visible to the player in the Readme file).